sectionproperties/analysis/cross_section.py

```diff
@@ -1309,21 +1404,25 @@ def get_ig(self):
 
         ::
 
-            section = CrossSection(geometry, mesh)
+            section = Section(geometry, mesh)
             section.calculate_geometric_properties()
             (ixx_g, iyy_g, ixy_g) = section.get_ig()
         """
 
-        return (self.section_props.ixx_g, self.section_props.iyy_g, self.section_props.ixy_g)
+        return (
+            self.section_props.ixx_g,
+            self.section_props.iyy_g,
+            self.section_props.ixy_g,
+        )
 
     def get_c(self):
         """
-        :return: Elastic centroid *(cx, cy)*
+        :return: Elastic centroid *()*
```

Comment:
I think this should be discarded/reverted, `:return: Elastic centroid *(cx, cy)*` seems correct here?

Reply:
Yep!